packages/calcite-components/src/components/functional/Heading.tsx

```diff
@@ -17,7 +17,7 @@ export const Heading: FunctionalComponent<HeadingProps> = (props, children): VNo
   delete props.level;
 
   return (
-    <HeadingTag class={props.class} key={props.key} level={props.level}>
```

Comment:
`level` prop doesn't seem to exist in the DOM?
was it used for styling or testing? if not, we should remove it

Reply:
I think this is fine to remove. @driskull Can you confirm?